consts/errors.js

```diff
@@ -115,9 +115,9 @@ const errors = {
     code: 'REVIEW_NOT_FOUND',
     message: 'Review with the specified id was not found.'
   },
-  REVIEW_NOT_CREATED: {
-    code: 'REVIEW_NOT_CREATED',
-    message: 'You are allowed to leave only one review for a tutor.'
+  REVIEW_ALREADY_EXISTS: {
+    code: 'REVIEW_ALREADY_EXISTS',
+    message: 'You are allowed to leave only one review.'
```

Comment:
maybe, try to specificate it better, for example: 
You allready left a review for this user with this offer, or smth like that

Reply:
remove